controller/errror.controller.js

```diff
@@ -0,0 +1,7 @@
+exports.handlePSQLErrors = (err, req, res, next) => {
+  if (err.code === "22P02") {
```

Comment:
Don't need this yet - no need for PSQL error handling yet - remove until you actually need it. Try to stick to ticket guide.